pkg/kinds/dashboard/dashboard_spec_gen.go

```diff
@@ -612,13 +612,13 @@ type RangeMap struct {
 	// Range to match against and the result to apply when the value is within the range
 	Options struct {
 		// Min value of the range. It can be null which means -Infinity
-		From float64 `json:"from"`
+		From *float32 `json:"from"`
```

Comment:
This is really weird, i don't know why it's happening, but it's not enough of a problem now to block this PR. We've gotta figure it out down in the code generators.